### PR TITLE
[DOCU-3051] Auto docs 3.3

### PR DESCRIPTION
**Pull request by Guaris**
https://konghq.atlassian.net/browse/DOCU-3051

Auto docs for EE and OSS.

### CODE REVIEW

File: app/_src/gateway/plugin-development/pdk/kong.request.md [hunk 1]
```diff
@@ -485,11 +485,17 @@ Returns the table of query arguments obtained from the query string.  Keys
  Note that a query string `?foo&bar` translates to two boolean `true`
  arguments, and `?foo=&bar=` translates to two string arguments containing
  empty strings.
-
+{% if_version lte 3.2 %}
```
Comment (Guaris) [inline app/_src/gateway/plugin-development/pdk/kong.request.md:488]:
```suggestion
{% if_version lte:3.2.x %}
```

File: app/_src/gateway/plugin-development/pdk/kong.request.md [hunk 2]
```diff
@@ -485,11 +485,17 @@ Returns the table of query arguments obtained from the query string.  Keys
...
  By default, this function returns up to **100** arguments. The optional
  `max_args` argument can be specified to customize this limit, but must be
  greater than **1** and not greater than **1000**.
-
+{% endif_version %}
+{% if_version gte 3.3 %}
```
Comment (Guaris) [inline app/_src/gateway/plugin-development/pdk/kong.request.md:493]:
```suggestion
{% if_version gte:3.3.x %}
```

File: app/_src/gateway/plugin-development/pdk/kong.response.md [hunk 3]
```diff
@@ -122,9 +122,17 @@ Returns a Lua table holding the response headers.  Keys are header names.
  headers as the client would see them upon reception, including headers
  added by Kong itself.
 
+{% if_version lte:3.2 %}
```
Comment (Guaris) [inline app/_src/gateway/plugin-development/pdk/kong.response.md:125]:
```suggestion
{% if_version lte:3.2.x %}
```

File: app/_src/gateway/plugin-development/pdk/kong.response.md [hunk 4]
```diff
@@ -122,9 +122,17 @@ Returns a Lua table holding the response headers.  Keys are header names.
...
  By default, this function returns up to **100** headers. The optional
  `max_headers` argument can be specified to customize this limit, but must
  be greater than **1** and equal to or less than **1000**.
+{% endif_version %}
+{% if_version lte:3.3 %}
```
Comment (Guaris) [inline app/_src/gateway/plugin-development/pdk/kong.response.md:130]:
```suggestion
{% if_version gte:3.3.x %}
```

File: app/_src/gateway/plugin-development/pdk/kong.tracing.md [hunk 5]
```diff
@@ -91,8 +91,12 @@ Adds an error event to a Span
 * **status** (`number`):  status code
 
 
-
+{% if_version gte:3.2.x %}
```
Comment (Guaris) [inline app/_src/gateway/plugin-development/pdk/kong.tracing.md:94]:
```suggestion
{% if_version lte:3.2.x %}
```

**Review by lena-larionova — APPROVED** [review]
Looks good to me! You mentioned that CLI will come separately, right?

**Review by lena-larionova — CHANGES_REQUESTED** [review]
Never mind, something's off. There's a bunch of [changes to kong.conf.default](https://github.com/Kong/kong-ee/commits/next/3.3.x.x/kong.conf.default) that I don't see reflected here. Did you build from OSS or Enterprise? 

Or maybe the build image has an old copy of the file. Though it would have to be pretty old - I went looking for `cluster_dp_labels`, which was [added March 30th](https://github.com/Kong/kong/pull/10471/files), and didn't see it in this PR.

**Comment by Guaris** [comment]
> Never mind, something's off. There's a bunch of [changes to kong.conf.default](https://github.com/Kong/kong-ee/commits/next/3.3.x.x/kong.conf.default) that I don't see reflected here. Did you build from OSS or Enterprise?
> 
> Or maybe the build image has an old copy of the file. Though it would have to be pretty old - for example, I went looking for `cluster_dp_labels`, which was [added March 30th](https://github.com/Kong/kong/pull/10471/files), and didn't see it in this PR.

Ready, added all of that stuff in. Checked the CLI and no changes in OSS and EE that affect docs.

File: app/_src/gateway/reference/configuration.md [hunk 6]
```diff
@@ -532,6 +591,49 @@ The SNI (Server Name Indication extension) to use for Vitals telemetry data.
...
+Labels are only compatible with hybrid mode deployments with Kong Konnect
+(SaaS), this configuration doesn't work with self-hosted deployments.
+
+Keys and values follow the AIP standards: https://kong-aip.netlify.app/aip/129/
+
+Example: `deployment:mycloud,region:us-east-1`
+
+Default: none
+
+---
```
Comment (lena-larionova) [inline app/_src/gateway/reference/configuration.md:635]: This entry appears twice. Looks like the issue is the source file though, that'll need fixing.

```suggestion
```

**Pull request merged**